HVZ.Persistence/Models/Game.cs

```diff
@@ -85,8 +84,20 @@ public HashSet<Player> ZombiesAndOzs
     /// The role to put new people in when they join this game
     /// </summary>
     public Player.gameRole DefaultRole { get; init; }
+    /// <summary>
+    /// List of game IDs for players interested in being selected as OZs
+    /// </summary>
+    public List<string> OzPool { get; init; }
```

Comment:
```suggestion
    public HashSet<string> OzPool { get; init; }
```
we aren't expecting duplicate entries so a set makes more sense

Reply:
we would need to convert this to a list to pick random entries anyway because you can't index `HashSet`

6 of one, half a dozen of the other

Reply:
I would prefer that it is a `HashSet` by default, so we know there aren't accidentally duplicate entries stored. We can use it as a list just for this method.